CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# 1.9.0-beta02 (August 2025)
+
+_Changes since 1.9.0-beta01_
+
+## Breaking Changes
+
+### Multiple Platforms
+
+- _(prerelease fix)_ All public APIs tagged with `ExperimentalMaterial3ExpressiveApi` or `ExperimentalMaterial3ComponentOverrideApi` have been removed, please stay with previous alpha version to continue enjoying these features. See [Google Jetpack changelog](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.4.0-beta01) [#2278](https://github.com/JetBrains/compose-multiplatform-core/pull/2278)
```

Comment:
@MatkovIvan, better to change it to something like:

please use the previous Material3 version explicitly to continue enjoying these features (See [Google Jetpack changelog](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.4.0-beta01)):

```
implementation("org.jetbrains.compose.material3:material3:1.9.0-beta01")
```

Please change it in the PR.

Reply:
> Please use the previous alpha version

It is better to have Material3 in the sentence, otherwise it can be read that users need to use the previous version of CMP plugin